schwifty/bank_registry/generated_de.json

```diff
@@ -43410,7 +43428,7 @@
     "bank_code": "87070024",
     "name": "Deutsche Bank",
     "short_name": "Deutsche Bank",
-    "bic": "DEUTDEDB881",
```

Comment:
Should we update the script to short by BIC code and make it idempotent?

Reply:
That might be a good idea in general. Either by domestic bank code or by BIC.